media/scss/base/base.scss

```diff
@@ -1,7 +1,61 @@
-.btn-primary { background-color: $blue; }
+.btn-primary {
+    background-color: $blue;
+}
+
+.zoot {
+    font: inherit;
+}
+
+.graph-listing-item {
+    margin-bottom: 1rem !important;
+}
+
+.graph-listing-item p {
+    margin-left: 1rem;
+}
+
+.loginbtn {
+    border: 3px solid $background  !important;
+    border-radius: 0.5rem;
+}
+
+.loginbtn:focus,
+.loginbtn:hover {
+    background-color: $background  !important;
+    border: 3px solid white !important;
+    color: white !important;
```

Comment:
`!important` should only be used as a last resort, and is rarely necessary. There are better ways to make a CSS rule take precedence, like making the selector more specific.

Reply:
Found where in the scss the original styling was so no problem